### PR TITLE
[qa] cleanup runflake8 and runisort

### PR DESCRIPTION
**Pull request by Rishabh2801**
[qa] cleanup runflake8 and runisort  https://github.com/openwisp/openwisp-users/issues/59
Closes https://github.com/openwisp/openwisp-users/issues/59

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
 coverage
 coveralls
-isort
-flake8
 openwisp-utils
+openwisp-utils[qa]>=0.2.1
```
Comment (nemesisdesign) [inline requirements-test.txt]: avoid changing the permissions to the file

File: .travis.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ branches:
 
 before_install:
   - pip install --no-cache-dir -U -r requirements-test.txt
-  - ./runflake8
+  - flake8
   - # TODO: remove after openwisp-utils > 0.2 is released
   - pip install https://github.com/openwisp/openwisp-utils/tarball/master#egg=openwisp_utils --upgrade
   - ./runcheckmigration
```
Comment (nemesisdesign) [inline .travis.yml]: avoid changing the permissions to the file

File: runisort [hunk 3]
```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 set -e
-isort --check-only --recursive --diff --skip migrations || exit 1
+isort --check-only --recursive --diff || exit 1
+
```
Comment (atb00ker) [inline runisort]: There is an extra space here. Last line needs to be an empty line. :smile:

Comment (Rishabh2801) [reply runisort]: Can you suggest me what should I write instead of updated and sorry about this.

Comment (Rishabh2801) [reply runisort]: Can I change it because I have squashed it?

Comment (Rishabh2801) [reply runisort]: ok done i changed commit messages relevantly and removed extra line

Comment (Rishabh2801) [reply runisort]: i did. Yeah you are write i had not think about it that way now i will keep this in mind. Thanks :)

Comment (Rishabh2801) [reply runisort]: ok did change the body. I thought i have to put fixes issue number.

**Pull request merged**